documents/Work/Zipper/orchestrator-design-2026-05.md

```
@@ -0,0 +1,76 @@
+
+Three generation modes dispatched by `Program.cs → SelectMode() → IGenerationMode → GenerationRunner.RunAsync()`:
+
+| Mode | Adapter | Generator | Output |
+|------|---------|-----------|--------|
+| Standard | `StandardMode` | `ParallelFileGenerator` | ZIP + Load File(s) |
+| Loadfile-Only | `LoadfileOnlyMode` | `LoadfileOnlyGenerator` | Load File + `_properties.json` |
+| Production Set | `ProductionSetMode` | `ProductionSetGenerator` | Directory tree + Load Files + Manifest |
+
+### Standard Pipeline (ParallelFileGenerator)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)